packages/admin/admin/src/alert/Alert.tsx

```diff
@@ -0,0 +1,125 @@
+import { Close } from "@comet/admin-icons";
+import { Alert as MuiAlert, AlertTitle, buttonClasses, IconButton, Theme, Typography } from "@mui/material";
+import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import clsx from "clsx";
+import * as React from "react";
+
+export interface AlertProps {
+    severity?: "info" | "warning" | "error" | "success";
+    title?: React.ReactNode;
+    children?: React.ReactNode;
+    disableCloseButton?: boolean;
+    onClose?: () => void;
+    button?: React.ReactNode;
+}
+
+export type AlertClassKey = "root" | "message" | "title" | "text" | "button" | "closeIcon" | "hasTitle";
+
+const styles = (theme: Theme) =>
+    createStyles<AlertClassKey, AlertProps>({
+        root: {
+            display: "flex",
+            alignItems: "center",
+            backgroundColor: theme.palette.background.paper,
+            borderRadius: 4,
+            boxShadow: "0px 0px 20px 0px rgba(0, 0, 0, 0.10)",
+            position: "relative",
+            padding: "10px 12px 10px 20px",
```

Comment:
Isn't 20px a common spacing? If I remember correctly, I used the non-20px values because I had to subtract fixed sizes or paddings from certain elements so that the padding adds up to 20px in the end

Reply:
I was rather talking about the 12px

Reply:
20px is `theme.spacing(4)`

Reply:
That's what I meant with having to subtract. The element probably had 8px padding, so I calculated 20-8 or else it would've been 8px padding effectively